collection-scripts/gather_network_logs

```diff
@@ -50,6 +71,18 @@ function gather_ovn_kubernetes_data {
       ((i++))
     done
   done
+}
+
+function gather_ovn_kubernetes_data {
+  sample_node=$(oc get no -o jsonpath='{.items[0].metadata.name}')
+  sample_node_zone=$(oc get node "${sample_node}" -o jsonpath='{.metadata.annotations.k8s\.ovn\.org/zone-name}')
+  if [ "${sample_node}" = "${sample_node_zone}" ]; then
+    echo "INFO: INTERCONNECT MODE"
+    gather_ovn_kubernetes_data_interconnect_mode
```

Comment:
Does this have an impact on CI jobs in terms of duration, size, etc.?. Is there any payload blocking job running this?. Probably you have discussed it but have you had measure the impact on must-gather output's size?.

I'm curious about this because we had a discussion recently about the must-gather pod's direct effect on other core components in master node due to the filling all the storage.

Reply:
It does, we discussed this in length with @tssurya and @sferich888 yesterday, we agreed they will cut down the log gathering for networking nodes which is one of the biggest consumer (a followup work here or in oc), but will keep DBs, which will get looked at more in the future.

Reply:
yes what @soltysh said. SDN team is tracking this via https://issues.redhat.com/browse/SDN-3507. We will stop logging some of the heavy things were are logging in the container by default and instead divert those extra logs into a separate folder on the host that will not be gathered as part of default MG.

Reply:
Sounds good to me, thanks @tssurya